common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -720,13 +720,13 @@ def test_video_component_stores_speed_correctly_for_multiple_videos(self):
         self.video.speed = '1.0'
 
         # open video "B"
-        self.course_nav.go_to_sequential('B')
+        self.course_nav.go_to_vertical('Test Vertical-1')
```

Comment:
@jcdyer: @nasthagiri and I determined that the original method was misleadingly named. This was intended to clarify them, though I obviously missed the corresponding comment in this instance.